Documentation/ReleaseNotes.md

```diff
@@ -22,6 +22,14 @@ MRTK now supports running Oculus Quest Headsets and Controllers using the native
 
 For instructions on how to deploy your device on the Oculus Quest using the new pipeline, see the [Oculus Quest Setup Guide](CrossPlatform/OculusQuestMRTK.md)
 
+**Teleport pointer animation, handling, and sound improvements**
+
+The teleport pointer now has improved animations and audio feedback. We also improved the handling of the teleport pointer so it handles smoother when transitioning from pointing at nearby surfaces to farther away surfaces.
```

Comment:
Extra credit here and also fine for a future thing, but if you have a video or gif to share that would be really neat

Reply:
are streamable links fine like this one? https://streamable.com/ta4bhv